src/test/components/TooltipMarker.test.js

```diff
@@ -367,6 +367,20 @@ describe('TooltipMarker', function() {
             },
           },
         ],
+        [
+          'PreferenceRead',
+          114.9,
+          {
+            type: 'PreferenceRead',
```

Comment:
should that be `Preference` or `PreferenceRead`?

My understanding is that we should have `type` as `Preference`, but the name above being `PreferenceRead` or `PreferenceWrite`. Is that it?